model_analyzer/triton/client.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
The model analyzer is provided under the Apache 2 license, are these headers compatible? Have you cleared this with NVIDIA legal?

Reply:
@dzier mentioned that we needed to switch to BSD3 for this in [another PR](https://github.com/triton-inference-server/model_analyzer/pull/7#discussion_r505006282).

Reply:
This project, in particular, was cleared to use Apache 2.0. Switching the license means involving NVIDIA legal, if for no other reason that they need to know things changed.